src/Components/CoolButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+
+
+class CoolButtons extends React.Component {
+  render() {
+    return (
+      <div>
+        {/* <button class="button is-rounded my-class is-danger is-small">Button 1</button>
+        <button class="button is-small is-success">Button 2</button> */}
+        <button class={this.props.className} href={this.props.href}>{this.props.children}</button>
```

Comment:
Remember! In react is className not class (you should have a warning from react)